auditlog/events/client.go

```diff
@@ -0,0 +1,89 @@
+package events
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strconv"
+
+	"github.com/workos-inc/workos-go/auditlog"
+)
+
+const (
+	eventsPath = "/events"
+)
+
+// List returns a list of events.
+func List(params auditlog.ListRequestParams) *Iter {
+	return &Iter{auditlog.GetIter(params, func(params auditlog.ListRequestParams) ([]interface{}, auditlog.ListMeta, error) {
+		list, err := FindAll(params)
+
+		ret := make([]interface{}, len(list.Data))
+		for i, v := range list.Data {
+			ret[i] = v
+		}
+
+		return ret, list.ListMeta, err
+	})}
+}
+
+// FindAll returns a paginated set of Audit Log entries matching the search
+// query.
+func FindAll(params auditlog.ListRequestParams) (auditlog.EventList, error) {
+	path := eventsPath
+	q := url.Values{}
+
+	q.Add("limit", strconv.Itoa(params.GetLimit()))
+	q.Add("after", params.GetStartingAfter())
```

Comment:
Since a string is always provided even if there is no `after` we should probably check it isn't empty.

```suggestion
	if params.GetStartingAfter() != "" {
		q.Add("after", params.GetStartingAfter())
	}
```